src/client/sandbox/node/element.js

```diff
@@ -184,15 +191,17 @@ export default class ElementSandbox extends SandboxBase {
 
             args[valueIndex] = 'off';
         }
-        else if (loweredAttr === 'target' && DomProcessor.isTagWithTargetAttr(tagName)) {
-            const currentTarget = nativeMethods.getAttribute.call(el, 'target');
+        else if (loweredAttr === 'target' && DomProcessor.isTagWithTargetAttr(tagName) ||
```

Comment:
Use `TARGET_ATTRS` from https://github.com/DevExpress/testcafe-hammerhead/pull/1664/files#diff-94b2df7ad47ba85d634a6c72c32343c0R17 here and below

```js
...
else if (TARGET_ATTRS.indexOf(loweredAttr) > -1 && DomProcessor.isTagWithTargetAttr(tagName))
```